genlib/bspnode.h

```diff
@@ -17,7 +17,6 @@
 
 #pragma once
 
-#include "genlib/paftl.h"
 #include "p2dpoly.h"
 
 // Binary Space Partition
```

Comment:
`line` should be renamed to m_line - you currently have a clash in `pickMidPointLine()` where you define a local variable named line that hides the member. Also worth checking if the data members can be made private and have getters (but not that important)